src/core/api/index.js

```diff
@@ -162,56 +162,6 @@ export function callApi({
     .then((response) => (schema ? normalize(response, schema) : response));
 }
 
-type SearchParams = {|
```

Comment:
I moved this to its own file because I was finding this file got really crowded and I wanted to search files for "search" to edit this API but had to find `core/api/index.js` instead. It's isolated in 1f4600bd744dd8e3c0f5a4563a516aae70e8434e so if you just want to review the feature change it's in the second commit: fb0f11167f3561763e330a2da53bbb0afe9480c7